html/elements/del.json

```diff
@@ -68,7 +68,8 @@
               },
               "safari_ios": "mirror",
               "samsunginternet_android": "mirror",
-              "webview_android": "mirror"
+              "webview_android": "mirror",
+              "webview_ios": "mirror"
```

Comment:
This causes this error:

```
 ✖ html.elements.del - Error → Basic support in webview_ios was declared implemented in a later version (3.2) than the following sub-feature(s):
        → html.elements.del.cite: 2
        → html.elements.del.datetime: 2
```

However, the same applies to Safari, which says `3` for the subfeature and `≤4` for the parent feature. So I'm not sure why it only fails for the subfeature?

Reply:
This is due to an issue with mirroring where the downstream browser became a non-range no matter what.  This will be resolved by #24155!